addons/point_of_sale/static/tests/tours/sales_report_tour.js

```diff
@@ -0,0 +1,26 @@
+import * as ProductScreen from "@point_of_sale/../tests/tours/utils/product_screen_util";
+import * as PaymentScreen from "@point_of_sale/../tests/tours/utils/payment_screen_util";
+import * as ReceiptScreen from "@point_of_sale/../tests/tours/utils/receipt_screen_util";
+import * as Chrome from "@point_of_sale/../tests/tours/utils/chrome_util";
+import * as Dialog from "@point_of_sale/../tests/tours/utils/dialog_util";
+
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("PosSalesReportUITour", {
+    test: true,
+    steps: () =>
+        [
+            Dialog.confirm("Open session"),
+            ProductScreen.clickDisplayedProduct("Desk Organizer", true, "1.0", "5.10"),
+            ProductScreen.clickPayButton(),
+            PaymentScreen.clickPaymentMethod("Cash"),
+            PaymentScreen.enterPaymentLineAmount("Cash", "5.1"),
+            PaymentScreen.clickValidate(),
+            ReceiptScreen.isShown(),
+            ReceiptScreen.clickNextOrder(),
+            Chrome.clickMenuOption("Close Register"),
+            Dialog.secondary("Daily Sale"),
```

Comment:
just add this one line, with `Dialog.confirm` instead of `Dialog.secondary` to an existing tour.

Reply:
Check for sales report download now moved to existing tour `CashClosingDetails`.